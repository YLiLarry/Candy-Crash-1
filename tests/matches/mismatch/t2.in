6 ..
swap 2 3 left
