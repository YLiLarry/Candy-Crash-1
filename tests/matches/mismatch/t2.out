
__2 __1 __0 __0 __2 __1 
__1 __0 __0 __2 __1 __2 
__2 __3 __2 __0 __2 __1 
__3 __2 __0 __0 __1 __2 
__2 __1 __2 __1 __2 __3 
__1 __2 __0 __2 __3 __2 

__2 __1 __0 __0 __2 __1 
__1 __0 __0 __2 __1 __2 
__2 __3 __0 __2 __2 __1 
__3 __2 __0 __0 __1 __2 
__2 __1 __2 __1 __2 __3 
__1 __2 __0 __2 __3 __2 

__2 __1 ___ __0 __2 __1 
__1 __0 ___ __2 __1 __2 
__2 __3 _v0 __2 __2 __1 
__3 __2 ___ __0 __1 __2 
__2 __1 __2 __1 __2 __3 
__1 __2 __0 __2 __3 __2 

__2 __1 __3 __0 __2 __1 
__1 __0 __2 __2 __1 __2 
__2 __3 __1 __2 __2 __1 
__3 __2 _v0 __0 __1 __2 
__2 __1 __2 __1 __2 __3 
__1 __2 __0 __2 __3 __2 
